model/src/main/java/org/openremote/model/security/KeyStoreService.java

```diff
@@ -14,72 +14,14 @@ public interface KeyStoreService extends ContainerService {
 	String OR_KEYSTORE_PASSWORD = "OR_KEYSTORE_PASSWORD";
 	Logger LOG = Logger.getLogger(KeyStoreService.class.getName());
 
-	KeyStore getKeyStore(KeyStoreType type);
-	void storeKeyStore(KeyStore keyStore, KeyStoreType type);
+	KeyStore getKeyStore();
+	void storeKeyStore(KeyStore keyStore);
+	KeyStore getTrustStore();
+	void storeTrustStore(KeyStore keyStore);
 	char[] getKeyStorePassword();
-
 	KeyManagerFactory getKeyManagerFactory(String preferredAlias) throws Exception;
```

Comment:
Not sure the intention of `preferredAlias` rather than just `alias`, if it is expected that a realm prefixed aliase be supplied then name the parameter something like `realmPrefixedAlias`